script.js

```diff
@@ -10,34 +13,45 @@ function computerSelection() {
     return randomComputerSelection;
 }
 
-function playRound() {
-    //let playerInput =  prompt("Choose Rock, Paper, or Scissors!");
-    //let playerSelection = playerInput.toLowerCase();
-    //let computersMove = computerSelection();
-
-    if (
+//plays a single round and produces round winner
+function playRound(playerSelection) {
+    let computersMove = computerSelection();
+        if (
         (playerSelection === choiceRock && computersMove === choiceScissors) ||
         (playerSelection === choiceScissors && computersMove === choicePaper) ||
         (playerSelection === choicePaper && computersMove === choiceRock)) {
-        return `You won! ${playerSelection.toUpperCase()} beats ${computersMove.toUpperCase()}.`;
+        document.querySelector("h2").innerHTML = `Player 1 wins! ${playerSelection.toUpperCase()} beats ${computersMove.toUpperCase()} `;
+        document.querySelector(".p1-score").textContent = (playerOneScore++) + 1;
     } else if (
         (playerSelection === choiceRock && computersMove === choicePaper) ||
         (playerSelection === choiceScissors && computersMove === choiceRock) ||
         (playerSelection === choicePaper && computersMove === choiceScissors)) {
-        return `You lost! ${computersMove.toUpperCase()} beats ${playerSelection.toUpperCase()}.`;  
+        document.querySelector("h2").innerHTML = `Computer wins! ${computersMove.toUpperCase()} beats ${playerSelection.toUpperCase()} `;
+        document.querySelector(".p2-score").textContent = (playerTwoScore++) + 1;
     } else if (playerSelection === computersMove) {
-        return "Tie game, Play again.";
+        document.querySelector("h2").innerHTML = `Draw Game! ${playerSelection.toUpperCase()} vs ${computersMove.toUpperCase()} `;
+}
+winnerOutcome();
 }
+
+//targets button content to give clickable feature
+var allButtons = document.querySelectorAll(".content");
+allButtons.forEach(buttons => {
+    buttons.addEventListener("click", getPlayerChoice); 
+});
+
+function getPlayerChoice(event){
+    let playerSelection = (event.target.id);
+    playRound(playerSelection);
 }
 
-function game() {
-    var resultsArray = [];
-    for (let i = 0; i < 5; i++) {
-        var result = playRound();
-        resultsArray.push(result);
-        console.log(result);
-    }
-    console.log(resultsArray);
+function winnerOutcome() {
+    let p1Score = playerOneScore;
+    let p2Score = playerTwoScore;
+    if (p1Score === 5) {
```

Comment:
The 5 here is what we call "hard coded". Meaning if we ever wanted to change how many rounds are in a game a developer would have to find all the places 5 is used and update them. Its not very good for maintainability. Instead let's refactor everywhere this 5 is used into a variable. Maybe call it something like `NUMBER_OF_ROUNDS_PER_GAME`